client/state/sites/blog-stickers/reducer.js

```diff
@@ -17,6 +17,18 @@ export const items = createReducer(
 				[ action.payload.blogId ]: action.payload.stickers,
 			} );
 		},
+		[ SITES_BLOG_STICKER_ADD ]: ( state, action ) => {
+			const { blogId, stickerName } = action.payload;
+
+			// If the blog already has this sticker, do nothing
+			if ( state[ blogId ] && includes( state[ blogId ], stickerName ) ) {
+				return state;
+			}
+
+			return merge( {}, state, {
```

Comment:
I don't think we benefit from [merge](https://lodash.com/docs/4.17.4#merge) as you have written it.  If you want to use merge then you can skip the concat because thats what merge does on its own.  So you would be able to write:

```js
return merge( {}, state, { [ blogId ]: [ stickerName ] } );
```

Personally I usually lean towards the `Object.assign` method because `merge` does a lot

```js
return {
   ...state,
   [ blogId ]: [ stickerName ].concat( state[ blogId ] || [] )
};
```